[0m11:58:16.399637 [debug] [MainThread]: Acquiring new vertica connection "_test"
[0m11:58:16.399859 [debug] [MainThread]: Acquiring new vertica connection "_test"
[0m11:58:16.400051 [debug] [MainThread]: Creating schema "_ReferenceKey(database='vmart', schema='test16657486961044905757_test_basic', identifier=None)"
[0m11:58:16.405739 [debug] [MainThread]: Using vertica connection "_test"
[0m11:58:16.405934 [debug] [MainThread]: On _test: BEGIN
[0m11:58:16.406056 [debug] [MainThread]: Opening a new connection, currently in state init
[0m11:58:23.720892 [debug] [MainThread]: Connection '_test' was properly closed.
